workshop/content/docs/standards.md

```diff
@@ -70,21 +73,26 @@ pygeoapi over time:
 
 #### OGC APIs supported by pygeoapi
 
-pygeoapi is an OGC API Reference Implemetnation and implements numerous OGC API standards.
+pygeoapi implements numerous OGC API standards and draft standards. In addition, it is a Reference Implementation for some of them. From OGC Compliance Testing Program Policies & Procedures 08-134r9:
```

Comment:
1. I think https://docs.ogc.org/pol/08-134r11.html is the latest P&P?
2. Can we link to the actual P&P URL in the text above

Reply:
@tomkralidis +1